etc/org.opencastproject.animate.impl.AnimateServiceImpl.cfg

```diff
@@ -3,5 +3,5 @@
 #synfig.path=synfig
 
 # Job load for an animation job
-# Default: 1.0
-#job.load.animate=1.0
+# Default: 0.1
```

Comment:
animate internally uses FFmpeg to encode a video file. 0.1 seems a bit low for that, even though it's just a single stream encoding.

Reply:
Suggestion?

Reply:
Whatever you and Greg settle for as default for encoding ;-)

Reply:
@gregorydlogan, which load value do you suggest?

Reply:
I would suggest 1.0, but I'm fine with 0.6.